edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/tests/catalog/DimCatalogIntegrationTest.java

```diff
@@ -98,7 +95,7 @@ static void afterAll() {
         bdrsServer.stop();
     }
 
-    @Test
+    //@Test
```

Comment:
Could it be just `@Disabled`?

Reply:
Unfortunatelly no, since this depends on environment variable being there and it still fails in initialization even if it's disabled.

Reply:
that's a bummer! LGTM the way it is then!

Reply:
Yeah it's ugly, it's marked as removal anyway i doubt that we will have a config for running this nightly